pip/flatpak-pip-generator

```diff
@@ -30,8 +30,6 @@ parser.add_argument('--requirements-file', '-r',
 parser.add_argument('--build-only', action='store_const',
                     dest='cleanup', const='all',
                     help='Clean up all files after build')
-parser.add_argument('--no-build-isolation', action='store_true', default=False
```

Comment:
I'd rather invert the option instead of removing it completely. Just in case if someone would need it.

Reply:
If it's imaginably desirable, then we should just drop the entire parameter and redesign the tool to support build isolation. It does not support it at the moment.

Reply:
Oh. Now I wonder how it even worked if build isolation was enabled by default.

Reply:
It works mostly by luck.

Reply:
Added parameter back but as reversed.